static/app/views/issueDetails/streamline/header/attachmentsBadge.tsx

```diff
@@ -50,7 +50,7 @@ export function AttachmentsBadge({group}: {group: Group}) {
         aria-label={t("View this issue's attachments")}
       >
         {hasManyAttachments
-          ? t('50+ Attachments')
+          ? tct('[count]+ Attachments', {count: attachments.attachments.length})
```

Comment:
I updated the count to `100` on the endpoint, but we won't have to update it again if it just uses the list length rather than guessing.